Tests/Unit/Helpers.swift

```diff
@@ -374,6 +374,10 @@ class MockFile {
 }
 
 class MockMessageInterfaceExtension: MiniAppMessageDelegate {
+    func closeMiniApp(withConfirmation: Bool, completionHandler: @escaping (Result<Bool, MASDKError>) -> Void) {
+        //TO-DO: implementation pending
```

Comment:
Please don't commit TODOs, and name it without the dash: `//TODO: ` so they can be caught by analyse tools ;)

Reply:
Updated the test case implementation hence the TODO's are now removed.